site/content/docs/v1.5/maintainers.md

```diff
@@ -16,6 +16,11 @@ Please be sure to also go through the guidance under the entire [Contribute](sta
 - There is no need to manually request reviewers: after the PR is created, all maintainers will be automatically added to the list (note: feel free to remove people if they are on PTO, etc).
 - Be familiar with the [lazy consensus](https://github.com/vmware-tanzu/velero/blob/v1.5.1/GOVERNANCE.md#lazy-consensus) policy for the project.
 
+Some tips for doing reviews:
+- There are some code standards and general guidelines we aim for, described here: https://velero.io/docs/main/code-standards/
+- For reviewing documentation, please follow these guidelines: https://velero.io/docs/main/style-guide/
```

Comment:
I was going to suggest that we link to the pages for this branch, but I think linking to the most up to date version of these docs is best :+1: 

I think it would be better to use links though rather than including the URLs directly. For example:
> For reviewing documentation, please follow [these guidelines](https://velero.io/docs/main/style-guide/)